src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/self-closing-comp */
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
```

Comment:
- [ ] It will be great to make this a class-based component. This is a major requirement for this project as you can see below ⬇️ 
![Screenshot 2022-11-29 at 11 12 09](https://user-images.githubusercontent.com/32361963/204474795-932fce6f-9a70-44c9-ba3f-41a14a4a78d5.png)